### PR TITLE
Fix client fails forever when namenode ipaddr changed(HADOOP-17068)

### PR DESCRIPTION
**Pull request by groobyming**
## Description
**Fix client fails forever when namenode ipaddr changed([HADOOP-17068](https://issues.apache.org/jira/browse/HADOOP-17068))**  

## Documentation

(x) No documentation is needed.

### CODE REVIEW

File: src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java [hunk 1]
```diff
@@ -1,1039 +0,0 @@
...
-    private static final Logger LOG = LoggerFactory.getLogger(AzureBlobFileSystemStore.class);
-
-    private AbfsClient client;
-    private URI uri;
-    private final UserGroupInformation userGroupInformation;
-    private final String userName;
-    private final String primaryUserGroup;
-    // See https://issues.apache.org/jira/browse/HADOOP-16479 for DATE_TIME_PATTERN modification
-    // TODO Drop the change (and the file) after upgrading to hadoop 3.3.0
-    private static final String DATE_TIME_PATTERN = "E, dd MMM yyyy HH:mm:ss z";
```
Comment (findepi) [inline src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java:109]: was https://issues.apache.org/jira/browse/HADOOP-16479 backported to 3.2.3?

Comment (groobyming) [reply src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java:109]: @findepi 
I will update the version to the latest 3.3.3，and run the test

File: src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java [hunk 2]
```diff
@@ -1,1039 +0,0 @@
...
-         */
-        @Override
-        public boolean equals(Object obj) {
-            if (!(obj instanceof FileStatus)) {
-                return false;
-            }
-
-            FileStatus other = (FileStatus) obj;
-
-            if (!this.getPath().equals(other.getPath())) {
```
[the remark this answers is not in the document]
Comment (groobyming) [reply src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java:991]: @findepi 
I will update the version to the latest 3.3.3，and run the test

**Comment by electrum** [comment]
Closing since this should be fixed in the latest version. Thanks for contributing to Trino.

**Pull request closed without merging**